tsconfig.json

```diff
@@ -4,14 +4,15 @@
     "declaration": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2018"],
```

Comment:
Another overdue cleanup: most modern versions of Node don’t need legacy codegen.